### ISSUE TITLE
Needed more documentation

### ISSUE THREAD

**Issue by PhaserEditor2D** [angelozerr/tern-jshint#3]
There is any small doc to describe how to use this plugin? I see in code there is a "jshint" query but I have not idea if there is any other parameter or the structure of the response.

**Comment by angelozerr** [comment]
@boniatillo-com yes I know, it works like https://github.com/angelozerr/tern-lint/wiki/Editors#other-editors

No time for the moment to write doc, any contribution are welcome!

**Comment by PhaserEditor2D** [comment]
Added to the list of the contributions I would like to do.

**Comment by angelozerr** [comment]
@boniatillo-com you can use [Tern Console](https://github.com/angelozerr/tern.java/wiki/Tern-Console)  to see request/response with tern server.

**Comment by PhaserEditor2D** [comment]
Yes I do it :) thanks
